.github/ISSUE_TEMPLATE/bug.yml

```
@@ -58,11 +58,8 @@ body:
   - type: textarea
     attributes:
       label: Your logs
-      description: Paste the Logs from running watchtower with the `--debug` option between the backticks.
-      value: |
-        ```
-
-        ```
+      description: Paste the logs from running watchtower with the `--debug` option.
+      render: shell
```

```suggestion
      render: text
```
The log files could be rendered incorrectly with the `shell` highlighting. The [`text`](https://github.com/github/linguist/blob/4ea1b93800c27ecb2221d4a9b19d8881e9197544/lib/linguist/languages.yml#L6577) language should render it as a simple code block.